recipes-crosswalk/crosswalk/crosswalk_20.50.533.12.bb

```diff
@@ -419,6 +448,7 @@ DEFAULT_CONFIGURATION = "\
 
 # Not available on Yocto, so we use
 # what comes bundled with Chromium.
+# -Duse_system_flac=1
```

Comment:
It sounds like a bug in Chromium GYP build. Will it be solved by GN build?

Are you OK we live with not using system flac in this PR and file an issue to track it?